src/index.test.js

```diff
@@ -109,11 +109,11 @@ const match = {
 const renderWithRouter = (component) => {
   const history = createMemoryHistory();
   return {
-    ...render(withIntlConfiguration(
+    ...withIntlConfiguration(
```

Comment:
Please avoid unnecessary spread syntax. You can also use `<MemoryRouter>` for convenience.

Reply:
Removed spread syntax and use `<MemoryRouter>`